Bottom: 2ec67e7f0d984db382c2cc20117762813f88342f
Top:    a2a227c85c2d77caa956fa73ecbf1a6a7c0add2e
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-07 09:54:34 +0000

IMPORTANT: DONT DRAW POUR OUTLINES DURING EXPORT

They can produce shorted boards otherwise!


---

diff --git a/src/draw.c b/src/draw.c
index a81ecae..989bd10 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -708,7 +708,8 @@ pour_callback (const BoxType * b, void *cl)
   struct pin_info *i = (struct pin_info *) cl;
   PourType *pour = (PourType *)b;
 
-  DrawPour (i->Layer, pour, 0);
+  if (gui->gui)
+    DrawPour (i->Layer, pour, 0);
 
   if (pour->PolygonN)
     {
